packages/terra-form/src/components/Control.jsx

```diff
@@ -0,0 +1,73 @@
+// eslint-disable react/jsx-boolean-value
+
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+
+import Input from './Input';
+
+const propTypes = {
+  /**
+   * The input type
+   */
+  type: PropTypes.oneOf(['checkbox', 'radio']).isRequired,
+  /**
+   * Additional attributes for the input object
+   */
+  attrs: PropTypes.object,
```

Comment:
Does anyone have strong opinions on this? Something like inputAttributes seems more clear to me